lavamoat/browserify/main/policy.json

```diff
@@ -2693,7 +2693,7 @@
         "@zxing/library>ts-custom-error": true
       }
     },
-    "extension-port-stream>readable-stream>abort-controller": {
+    "@lavamoat/lavapack>readable-stream>abort-controller": {
```

Comment:
None of these browserify policies should have changed, so let's explore why these changes are trivial:
- the path to abort-controller changed because packages got rearranged in the tree and this is now the shortest path to it.
- readable-stream got deduplicated into a newer version that uses AbortSignal directly etc.